### DISCUSSION TITLE
General

### DISCUSSION THREAD

**Discussion by StrangeGirlMurph** [StrangeGirlMurph/obsidian-wikipedia-helper#4] [General]
Lets just talk! Tell me what you think about the plugin :)

**Comment by lhansford** [comment]
Hey @StrangeGirlMurph, thanks for making this plugin! I'm just using it in a basic way to quickly link text to a wikipedia article, but it's working perfectly for that!

**Reply by StrangeGirlMurph** [reply]
Happy to hear ☺️ Me too :) Basic linking is all I need really. I mean the advanced features are cool and really useful if you have that kind of workflow but I guess only a fraction of people actually use them.
